packages/@aws-cdk/aws-ecs/lib/log-drivers/splunk-log-driver.ts

```diff
@@ -0,0 +1,146 @@
+import { Construct } from '@aws-cdk/core';
+import { ContainerDefinition } from '../container-definition';
+import { BaseLogDriverProps } from './base-log-driver';
+import { LogDriver, LogDriverConfig } from "./log-driver";
+import { ensureInList, ensureInRange, stringifyOptions } from './utils';
+
+/**
+ * Specifies the splunk log driver configuration options.
+ *
+ * [Source](https://docs.docker.com/config/containers/logging/splunk/)
+ */
+export interface SplunkLogDriverProps extends BaseLogDriverProps {
+  /**
+   * Splunk HTTP Event Collector token.
+   */
+  readonly token: string;
```

Comment:
Is this token a secret? If so, type it as a `core.Secret`.

Reply:
Updated.